lib/widgets/end_drawer/junto_account.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:junto_beta_mobile/backend/backend.dart';
+import 'package:junto_beta_mobile/widgets/dialogs/delete_account_dialog.dart';
+
+import 'junto_account_appbar.dart';
```

Comment:
Avoid file mixing packages and file imports. Dart will treat these as separate types which can cause problems

Reply:
This rule doesn't apply here. Within the same lib both imports work https://dart.dev/guides/language/effective-dart/usage#do-use-relative-paths-when-importing-libraries-within-your-own-packages-lib-directory